cccatalog-api/cccatalog/urls.py

```diff
@@ -73,31 +76,37 @@
 
 urlpatterns = [
     path('', schema_view.with_ui('redoc', cache_timeout=None), name='root'),
+    path('v1/', schema_view.with_ui('redoc', cache_timeout=None), name='root'),
     path('admin/', admin.site.urls),
-    path('oauth2/register', Register.as_view(), name='register'),
-    path('oauth2/key_info', CheckRates.as_view(), name='key_info'),
+    path('v1/oauth2/register', Register.as_view(), name='register'),
+    path('v1/rate_limit', CheckRates.as_view(), name='key_info'),
     path(
-        'oauth2/verify/<str:code>',
+        'v1/oauth2/verify/<str:code>',
         VerifyEmail.as_view(),
         name='verify-email'
     ),
     re_path(
-        r'^oauth2/',
+        r'v1/oauth2/',
         include('oauth2_provider.urls', namespace='oauth2_provider')
     ),
     # path('list', CreateList.as_view()),
     # path('list/<str:slug>', ListDetail.as_view(), name='list-detail'),
-    re_path('image/search', SearchImages.as_view()),
-    path('image/browse/<str:provider>', BrowseImages.as_view()),
-    path('image/<str:identifier>', ImageDetail.as_view(), name='image-detail'),
     path(
-        'image/related/<str:identifier>',
+        'v1/images/<str:identifier>', ImageDetail.as_view(), name='image-detail'
+    ),
+    re_path('v1/images', SearchImages.as_view()),
```

Comment:
I've wrapped it with an `include` now, which makes things a bit better. I would like to use a router in the future but it seems like it wouldn't be a good fit for us at the moment because we haven't set up any View Sets.